File: src/ZKWeb.MVVMPlugins/MVVM.Angular.Website/static/src/modules/generated_module/dtos/tenant-input-dto.ts
/** 租户传入信息 */
export class TenantInputDto {
	/** 租户Id */
	public Id: string;
	/** 租户名称 */
	public Name: string;
	/** 超级管理员名称 */
	public SuperAdminName: string;
	/** 超级管理员密码 */
	public SuperAdminPassword: string;
	/** 超级管理员确认密码 */
	public SuperAdminConfirmPassword: string;
	/** 备注 */
	public Remark: string;
}

File: src/ZKWeb.MVVMPlugins/MVVM.Common.MultiTenant/src/Application/Dtos/TenantInputDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.Text;
using ZKWeb.MVVMPlugins.MVVM.Common.Base.src.Application.Dtos;

namespace ZKWeb.MVVMPlugins.MVVM.Common.MultiTenant.src.Application.Dtos
{
    [Description("租户传入信息")]
    public class TenantInputDto : IInputDto
    {
        [Description("租户Id")]
        public Guid Id { get; set; }
        [Description("租户名称")]
        public string Name { get; set; }
        [Description("超级管理员名称"), Required]
        public string SuperAdminName { get; set; }
        [Description("超级管理员密码"), StringLength(int.MaxValue, MinimumLength = 6)]
        public string SuperAdminPassword { get; set; }
        [Description("超级管理员确认密码"), StringLength(int.MaxValue, MinimumLength = 6)]
        public string SuperAdminConfirmPassword { get; set; }
        [Description("备注")]
        public string Remark { get; set; }
    }
}
